woocommerce/woocommerce-android
# Group analytics hub information section

Partially closes: #8130

### Description
This PR groups analytics hub section's information to improve the talkback experience when using the app. It does that by introducing some minor changes in the information view layout.

### Testing instructions
1. Enable Talkback on the device settings
2. Go to the app and open the MyStrore screen
3. Tap on the See more button in the store stats section
4. Tap on different information sections and check that they are read as a group


### Images/gif

<img width="300" src="https://user-images.githubusercontent.com/18119390/223835313-72d41732-8aca-44ee-be0c-cc3c0e40dfae.png" />
